### DISCUSSION TITLE
[Invalid] Captcha on Contact form

### DISCUSSION THREAD

**Discussion by PikaMug** [NamelessMC/Nameless#2543] [Q&A] [answered]
I receive daily spam emails about "investment opportunities" and "cheap SEO" through Contact. It would be great to have a captcha that must be completed before submission.

**Comment by partydragen** [comment, marked as answer]
You can already by setting up captcha 

on StaffCP -> General Settings -> Registrations there you can setup the captcha

**Reply by PikaMug** [reply]
> You can already by setting up captcha
> 
> on StaffCP -> General Settings -> Registrations there you can setup the captcha

Yes, but does this apply to the Contact page?

**Reply by partydragen** [reply]
Yes but it also applies to the register page

**Reply by PikaMug** [reply]
Looks like I thought I had captcha enabled, but in fact did not. Thanks for your help.
